src/lib/server/rateLimits.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
+import { RATE_LIMITS, RATE_LIMIT } from "$env/static/private";
+import JSON5 from "json5";
+
+// RATE_LIMIT is the legacy way to define messages per minute limit
+export const rateLimitsSchema = z
```

Comment:
It's not `rateLimits` except for the `messagesPerMinute`

It's more like `USAGE_LIMITS`

Reply:
fixed as well!